lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -534,26 +534,18 @@ def set_zsort(self, zsort):
 
         Parameters
         ----------
-        zsort : bool or {'average', 'min', 'max'}
-            For 'average', 'min', 'max' the z-order is determined by applying
-            the function to the z-coordinates of the vertices in the viewer's
-            coordinate system. *True* is equivalent to 'average'.
+        zsort : {'average', 'min', 'max'}
+            The function applied on the z-coordinates of the vertices in the
+            viewer's coordinate system, to determine the z-order.  *True* is
+            deprecated and equivalent to 'average'.
         """
-
         if zsort is True:
+            cbook.warn_deprecated(
+                "3.1", "Passing True to mean 'average' for set_zsort is "
+                "deprecated; pass 'average' instead.")
```

Comment:
We should decide when we will remove this. I don't really have a feel for how prevalent this is

Reply:
3.3 would be normal.